app/src/main/res/layout-sw600dp-land/fragment_settings.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/parent_layout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:theme="@style/SettingsTheme">
+
+    <com.willowtree.vocable.customviews.VocableImageButton
+        android:id="@+id/settings_close_button"
+        style="@style/CloseButton"
+        android:layout_width="@dimen/settings_close_button_width"
+        android:layout_height="@dimen/settings_close_button_height"
+        android:layout_marginStart="@dimen/settings_close_button_start_margin"
+        android:layout_marginTop="@dimen/settings_close_button_top_margin"
+        android:scaleType="center"
+        android:src="@drawable/close_action_button_icon"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/settings_title"
+        style="@style/SettingsTitle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/settings"
+        app:layout_constraintBottom_toBottomOf="@+id/settings_close_button"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="@+id/settings_close_button"
+        app:layout_constraintVertical_chainStyle="packed" />
+
+    <include
+        android:id="@+id/settings_options_container"
+        layout="@layout/settings_options_layout"
+        android:layout_width="match_parent"
```

Comment:
It's recommended that, whenever you wanna use `match_parent` in a `ConstraintLayout`, that we set the start/end to `parent` and give a width of 0dp (MATCH_CONSTRAINT).

To prove I'm not lying, it's in the [docs](https://developer.android.com/reference/android/support/constraint/ConstraintLayout#wrap_content-:-enforcing-constraints--added-in-1.1-) as a note